bucket/7zip.json

```diff
@@ -48,6 +58,9 @@
             },
             "32bit": {
                 "url": "https://7-zip.org/a/7z$cleanVersion.msi"
+            },
+            "arm64": {
+                "url": "https://www.7-zip.org/a/7z$cleanVersion-arm64.exe"
```

Comment:
So the `.exe` can be processed with the same logic the `.msi` is processed with?

Drop the `www.` to be consistent?

Reply:
The `.exe` file is handled by `pre_install` script while `.msi` is extracted.

Hmm, should `www` be added to all the URLs? The website in https://sourceforge.net/projects/sevenzip/ has `www`.